### PR TITLE
feat: :sparkles: use `tqdm` instead of `fastprogress` in reference scripts

### PR DESCRIPTION
**Pull request by odulcy-mindee**
I have not been able to run `train_pytorch.py` script on Colab. An idea is to use `tqdm` instead of `fastprogress`.

Related to https://github.com/mindee/doctr/discussions/1388 and https://github.com/mindee/doctr/discussions/1106

### CODE REVIEW

**Review by felixdittrich92 — CHANGES_REQUESTED** [review]
Thanks @odulcy-mindee lgtm :)
Only one thing if we are onces on it could you extend the progress bar also to each evaluation loop ?

File: references/classification/train_pytorch.py [hunk 1]
```diff
@@ -329,9 +330,9 @@ def main(args):
     # Create loss queue
     min_loss = np.inf
     # Training loop
-    mb = master_bar(range(args.epochs))
+    mb = trange(args.epochs)
```
Comment (felixT2K) [inline references/classification/train_pytorch.py:333]: Can be removed ?

File: references/classification/train_tensorflow.py [hunk 2]
```diff
@@ -293,9 +294,9 @@ def main(args):
     min_loss = np.inf
 
     # Training loop
-    mb = master_bar(range(args.epochs))
+    mb = trange(args.epochs)
```
Comment (felixT2K) [inline references/classification/train_tensorflow.py:297]: same and so on :)

File: references/classification/train_pytorch.py (hunk 1 above)
Comment (odulcy-mindee) [reply references/classification/train_pytorch.py:333]: Yeah, it's not that useful

**Review by felixT2K — APPROVED** [review]
Thanks :)

**Review by felixdittrich92 — APPROVED** [review]

**Pull request merged**